src/App.jsx

```diff
@@ -0,0 +1,55 @@
+// import logo from './logo.svg';
+import React from 'react';
+import './App.css';
+import styled, { createGlobalStyle } from 'styled-components';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import Body from './components/Body';
+
+const stylingVariables = {
+  bgColor: '#fffcde',
+  black: '#121212',
+  gray: '#545353',
+  white: '#f7f7f7',
+};
+
+export const styleContext = React.createContext();
+
+const GlobalStyle = createGlobalStyle`
+  :root {
+    font-size:16px;
+  }
+
+  @media screen and (max-width: 480px) {
+    :root {
+      font-size:14px;
```

Comment:
this looks really bad when you change the screens. So you have to have lots of media queries to changes the `font-size` for different screens.  Simply use `em` and you REDUCE this variability.